commons/data_access_layer/cosmos_db.py

```diff
@@ -222,17 +222,8 @@ def find(
         function_mapper = self.get_mapper_or_dict(mapper)
         return function_mapper(self.check_visibility(found_item, visible_only))
 
-    def find_all(
```

Comment:
We should latter discuss about formatting. I left a comment only as a reminder of it.

Reply:
fixed up